index.html

```diff
@@ -444,8 +444,8 @@ <h2><dfn>observe()</dfn> method</h2>
             <li>If <var>options</var>'s <a>buffered</a> flag is set:
               <ol>
                 <li>Let <var>entries</var> be the <a>PerformanceEntryList</a>
-                object returned by the <a href="#filter-buffer-by-name-and-type">
-                </a> algorithm with <var>buffer</var> set to <a>relevant
+                object returned by <a href="#filter-buffer-by-name-and-type"></a>
```

Comment:
Temporary fix... will file a ReSpec bug. 

```suggestion
                object returned by <a href="#filter-buffer-by-name-and-type">§ Filter buffer by name and type</a>
```